test/NameBinding/import-resolution-overload_swift5.swift

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t %S/Inputs/overload_vars.swift
+// RUN: %target-typecheck-verify-swift -swift-version 5 -I %t
+
+import overload_vars
+
+func useString(_ str: String) {}
+
+// In Swift 5, we currently consistently reject cross-module property overloading.
```

Comment:
You should have test errors that expect errors in that case

Reply:
Thanks Slava, I've gone ahead and made that change.

Reply:
Did you not push it yet?

Reply:
I did push, and it shows up on GitHub for me (HEAD at https://github.com/apple/swift/pull/18488/commits/c54ab4d75e24f56089e001fce81b9526a8c2e4e3). Should I re-rebase and try to push again?

Reply:
I'm looking at this part:

```
// In Swift 5, we currently consistently reject cross-module property overloading.
 // FIX-ME: This seems reasonable to permit.
```

Is there a test for this behavior?

Reply:
@slavapestov Apologies, I realise now that the comment isn't particularly well-worded. When I say "reject cross-module property overloading", what I really mean is "properties from this module always shadow properties from the other module, and therefore the other module's properties never show up in the overload set" – which is what the tests in that file are testing.

Your original comment prompted me to add expected-errors in order to ensure the other module's properties didn't show up in the overload set (previously I was just testing that a bare reference wasn't ambiguous).

I'll edit the comment in the test file to clarify things.